build.gradle.kts

```diff
@@ -11,7 +11,7 @@ plugins {
 
 allprojects {
   group = "uk.gov.justice.service.hmpps"
-  version = "1.2.1"
+  version = "1.3.0-beta"
```

Comment:
I've tested this locally on other projects and have reasonable confidence it hasn't broken everything ever. However I'd like to see it working in the wild before releasing a full version, messing with Auth is risky.